public/config/configuration.json

```diff
@@ -6,5 +6,8 @@
   "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "KEYCLOAK_CONFIG_URL": "/local-keycloak-config-url/keycloak.json",
   "ADDRESS_COMPLETE_KEY": "RH99-RN99-GB72-FW86",
-  "LD_CLIENT_ID": "5e5d5af4574f7e09ab53d03b"
+  "VUE_APP_PAY_ROOT_API": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1",
```

Comment:
I think it's ok to rename these if that's the intent, but I don't think it's ok to have duplicate keys (tech debt). Please clean up one way or the other.

Reply:
Reduced to the single property, will look at merging/renaming as a dept-ticket in the (very) near future